README.md

```diff
@@ -1,45 +1,57 @@
+# WAZP 🐝
+**W**asp **A**nimal-tracking **Z**oo project with **P**ose estimation
+(name is subject to refinement)
 
+## Overview
 
-## How to contribute
-### Setup
-* Clone the repository and install the package in editable mode (including all `dev` dependencies):
+WAZP is a dashboard built with [Dash-Plotly](https://dash.plotly.com/) for analysing animal tracking data. It is designed to work with pose estimation output from [DeepLabCut](https://github.com/DeepLabCut/DeepLabCut).
 
-```bash
+The package is currently in early development 🏗️ and is not yet ready for use. Stay tuned ⌛
+
+## Installation
+
+It is recommended to install WAZP inside a [conda](https://docs.conda.io/en/latest/) environment.
+Once you have `conda` installed, the following commands will create and activate a suitable environment, named `wazp-env`.
+
+```sh
+conda create -n wazp-env -c conda-forge python=3 pytables
+conda activate wazp-env
+```
+
+Next clone the GitHub repository, navigate to the local repository folder, and install WAZP.
+
+```sh
 git clone https://github.com/SainsburyWellcomeCentre/WAZP
 cd WAZP
-pip install -e '.[dev]'
+pip install .
 ```
-* Initialize the pre-commit hooks:
 
-    ```bash
-    pre-commit install
-    ```
-### Workflow
-* Create a new branch for your changes, make your changes and commit them.
+## Launching the dashboard
+
+You can launch the dashboard by opening a terminal and running the following command from the root of the repository:
 
-* Push your changes to GitHub and open a draft pull request.
-* If all checks run successfully, you may mark the pull request as ready for review.
-* When your pull request is approved, squash-merge it into the `main` branch and delete the feature branch.
+```sh
+python wazp/app.py
+```
 
-### How to run tests and type checks locally
-For debugging purposes, you may also want to run the tests and the type checks locally, before pushing. This can be done with the following commands:
+Alternatively, on Linux and MacOS, you can also run the following command:
 
-```bash
-cd WAZP
-pytest
-mypy -p wazp
+```sh
+sh start_wazp_server.sh
 ```
-### Versioning and deployment
-The package is deployed to PyPI automatically when a new release is created on GitHub. The version number is automatically determined from the latest tag on the `main` branch.
 
-Details on how this is handled with `setuptools_scm` **TBD**.
+Both commands will launch a local web server. If the dashboard does not automatically open in your default browser, click the link in the terminal to open it (the link will be of the form `http://localhost:8050/`).
 
-### License
+## License
 
-⚖️ [BSD 3-Clause](./LICENSE)
+⚖️ [BSD 3-Clause](https://opensource.org/licenses/BSD-3-Clause)
 
-### Template
+## Template
 This package layout and configuration (including pre-commit hooks and GitHub actions) have been copied from the [python-cookiecutter](https://github.com/SainsburyWellcomeCentre/python-cookiecutter) template.
```

Comment:
(Soft suggestion) Perhaps this is better in CONTRIBUTING?

Reply:
You are right, I moved it now